pom.xml

```diff
@@ -99,9 +99,7 @@
         <spring-statemachine.version>3.2.0</spring-statemachine.version>
         <swagger-annotations.version>2.2.8</swagger-annotations.version>
         <springdoc-openapi.version>1.6.14</springdoc-openapi.version>
-        <wultra-rest-client.version>1.6.0</wultra-rest-client.version>
```

Comment:
@romanstrobl May we use only single version? Lime core modules are not released separately.

Reply:
Yes we can.